src/Sylius/Bundle/OrderBundle/Tests/DependencyInjection/Compiler/TagCartContextsPassTest.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace DependencyInjection\Compiler;
+
+use Matthias\SymfonyDependencyInjectionTest\PhpUnit\AbstractCompilerPassTestCase;
+use Matthias\SymfonyDependencyInjectionTest\PhpUnit\DefinitionHasTagConstraint;
+use Sylius\Bundle\OrderBundle\DependencyInjection\Compiler\RegisterCartContextsPass;
+use Sylius\Bundle\OrderBundle\DependencyInjection\Compiler\TagCartContextsPass;
+use Sylius\Component\Order\Context\CartContextInterface;
+use Sylius\Component\Order\Context\ResettingCartContextInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+
+final class TagCartContextsPassTest extends AbstractCompilerPassTestCase
+{
+
+    /**
+     * @test
+     */
```

Comment:
```suggestion
    /** @test */
```
Small change to keep all tests consistent :). Applies to all test methods.